# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

HeadActionGoal action_goal
HeadActionResult action_result
HeadActionFeedback action_feedback
